src/main/java/net/snowflake/client/loader/LoadResultListener.java

```diff
@@ -69,7 +69,7 @@
    *
    * @return the number of rows in errors
    */
-  int getErrorRecordCount();
+  long getErrorRecordCount();
```

Comment:
oh, this change is a breaking change. Add a new method `long getErrorRecordCountInLong()`

Reply:
We need to keep the existing `int` based methods not to break the existing application. If the customer needs the get the long, they can update their code calling `long` based methods.

Reply:
ah ok, sounds good. Can have both in there!